<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-ERHSHJS" failures="0" tests="2" name="Cucmber.TestNGTestRunner" time="41.137" errors="1" timestamp="2024-06-20T12:43:03 SAST" skipped="0">
  <testcase classname="Cucmber.TestNGTestRunner" name="runScenario" time="19.139"/>
  <testcase classname="Cucmber.TestNGTestRunner" name="runScenario" time="21.998">
    <error type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertFalse(Assert.java:34)
at junit.framework.Assert.assertFalse(Assert.java:41)
at AutomationAcademy.stepDefinitation.StepDefinitationImpl.checkout_and_submit_the_order(StepDefinitationImpl.java:50)
at âœ½.CheckOut ZARA COAT 3 and submit the order(file:///C:/Users/MOBASSIR%20AHMAD/workspace-selenium/CucumberFramework/src/test/java/Cucmber/SumbitOrder.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Cucmber.TestNGTestRunner -->
